modules/node-management.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * architecture/architecture.adoc
+
+[id='node-management-{context}']
+= Node management in fully-managed {product-title}
+
+The fully-managed version of {product-title} version 4.0 integrates management of
+the container operating system and cluster management. Because the cluster manages
+both its updates and updates to RHCOS, {product-title} provides an opinionated
+lifecycle management experience that simplifies the orchestration of node upgrades.
+
+{product-title} employs three DaemonSets and controllers to simplify node management:
```

Comment:
Going into details too fast - needs at least 2-3 para and a diagram that summarizes the whole lifecycle.  Also, same comment here about operators vs controllers.  

This is the signature part of v4 and needs a “ELI5” intro and a really strong visual flow so people can grasp it.

The subsequent paras are way too deep, ignition needs to be introduced at a really high level.

Others can provide more context but I might say:

1. Nodes run an operating system designed to be configured and updated as part of a larger system (RHCoS)
2. On boot, nodes have a component Ignition that asks the cluster for the appropriate configuration and software version
3. After boot, the Kubelet starts and joins the node to the cluster
4. The configuration of the machine is managed by the cluster via pods that run on each node and continuously reconcile the configuration, including software version, with the cluster

A bunch more details but would include how an update works, how a config change works, how that affects workloads, and how people are expected to interact with nodes.

We also should put the caveats about rhcos vs rhel on this page.

Reply:
@smarterclayton imo i'd add and expand the steps 1 to 8 from https://github.com/derekwaynecarr/openshift-the-easy-way/blob/initial-structure/demo/demo.md . It does create a full picture of what is happening and that will help with self troubleshooting